servicenow/help.md

```diff
@@ -592,6 +592,40 @@ Example output:
 
 ### Triggers
 
+#### Incident Created
+
+This trigger identifies if a new incident has been created.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|interval|integer|5|True|How often to poll for new incidents (in seconds)|None|5|
+|query|string|None|False|Non-encoded query string to match new incident records (will poll for any new incident if query is omitted)|None|short_description=Newbug|
+
+Example input:
+
+```
+{
+  "interval": 5,
+  "query": "short_description=Newbug"
```

Comment:
If there is a more complicated query with quotes we should test and document what that looks like

E.g.

"query": "short_description='Customer Defect'"